native/acter/src/api/timeline/msg_content.rs

```diff
@@ -0,0 +1,452 @@
+use acter_core::events::{
```

Comment:
skipping as I assume this is just a code-move. If there is logic changes to review, please point them out and I will review them.